ci/pipeline.yml

```diff
@@ -375,6 +404,44 @@ jobs:
         silent: true
         icon_emoji: ":concourse:"
         username: concourse-ci
+- name: jdk14-build
+  serial: true
+  public: true
+  plan:
+    - get: spring-boot-jdk14-ci-image
+    - get: git-repo
+      trigger: false
```

Comment:
This needs to be set to true - I was unsure if I should do that from the start. Let me know if I should change it already.